docs/state.md

```diff
@@ -1,6 +1,6 @@
 # State
 
-The output of a component is based on input properties passed from its parent as attributes. However, a component may also maintain internal state that it uses to control its view. If Marko detects a change to either input or to the internal state, the view will automatically be updated.
+The output of a component is based on input properties passed from it's parent as attributes. However, a component may also maintain internal state that it uses to control its view. If Marko detects a change to either input or to the internal state, the view will automatically be updated.
```

Comment:
I think the previous grammar was actually correct here. I'm fairly confident that the correct usage here should be `"its"` instead of `"it's"`.

Reply:
You are right! I reverted the change.